nameserver/server.go

```diff
@@ -215,70 +215,49 @@ func (s *DNSServer) Stop() error {
 }
 
 func (s *DNSServer) queryHandler(proto dnsProtocol) dns.HandlerFunc {
```

Comment:
Maybe we could go further and rename these functions for better names (eg, `queryHandler`? all these methods are going to handle queries...). I would suggest something like: `queryHandler` -> `localAHandler`, `rdnsHandler` -> `localPTRHandler`, `commonQueryHandler` -> `localHandler`